packages/mobile/src/common/enums/date/date-format.ts

```diff
@@ -0,0 +1,6 @@
+enum DateFormat {
+  dateTime = 'DD.MM.YYYY HH:mm a',
```

Comment:
Why is `a`? We don't seem to need AP/PM?

Reply:
it was in design, removed.